src/sentry/static/sentry/app/data/forms/organizationGeneralSettings.jsx

```diff
@@ -0,0 +1,165 @@
+import _ from 'lodash';
+import {extractMultilineFields} from '../../utils';
+
+const forms = [
+  {
+    // Form "section"/"panel"
+    title: 'General',
+    fields: [
+      {
+        name: 'name',
+        type: 'string',
```

Comment:
could we use a class as an enum here? so we can use formType.string?  would anything get intelligence or lint errors?

Reply:
Going to leave these as is, I think `type` is something that we will want from json schema in future